File: src/main/java/it/unibo/crabinv/Model/entities/enemies/EnemyType.java
package it.unibo.crabinv.Model.entities.enemies;

public enum EnemyType {
    SERVANT (10);

    private final int rewardForKill;

    EnemyType( int rewardForKill) {
        this.rewardForKill = rewardForKill;
    }

    public int getRewardForKill() {
        return rewardForKill;
    }

}

File: src/main/java/it/unibo/crabinv/model/entities/enemies/EnemyType.kt
package it.unibo.crabinv.model.entities.enemies

/**
 * It's the identifier of the enemy.
 * @param rewardForKill the reward for the death of the enemy
 */
enum class EnemyType(
    /**
     * @return the actual value for the reward
     */
    val rewardForKill: Int,
) {
    SERVANT(10),
}
